airsonic-main/src/test/java/org/airsonic/player/service/MediaScannerServiceTestCase.java

```diff
@@ -161,12 +153,15 @@ public void testScanLibrary() {
 
     @Test
     public void testSpecialCharactersInFilename() throws Exception {
-        Resource resource = resourceLoader.getResource("MEDIAS/piano.mp3");
+        InputStream resource = MediaScannerServiceTestCase.class
+                .getClassLoader()
+                .getResourceAsStream("MEDIAS/piano.mp3");
+        assert resource != null;
         String directoryName = "Muff1nman\u2019s \uFF0FMusic";
         String fileName = "Muff1nman\u2019s\uFF0FPiano.mp3";
         File artistDir = temporaryFolder.newFolder(directoryName);
         File musicFile = artistDir.toPath().resolve(fileName).toFile();
-        IOUtils.copy(resource.getInputStream(), new FileOutputStream(musicFile));
+        IOUtils.copy(resource, new FileOutputStream(musicFile));
```

Comment:
Maybe just use Google's Resources class, much shorter, and won't need that assertion:
```
Resources.copy(Resources.getResource("MEDIAS/piano.mp3"), new FileOutputStream(musicFile));
```

Reply:
Separate PR.